### ISSUE TITLE
Feature request: Separate backup files location

### ISSUE THREAD

**Issue by intertreeJK** [cardano-community/guild-operators#1632]
Suggestion: 

Script updates so that the previous versions of the files being updated can be copied into an explicit 'archive' subdirectory off of 'scripts'.

As the file lists can get a bit messy all in the same place as the current versions over time. 

Would be nice to have a single directory in which they can be stored/nuked from.

**Comment by Scitz0** [comment]
Not a bad idea. A simple wildcard delete command will remove them all though if you want to clear all backups.
`rm *_bkp*`

**Comment by intertreeJK** [comment]
Indeed. I am aware of this, but it still leaves the tidying up to the user,
instead of being nicely managed from the start to avoid root scripts
directory clutter.

**Comment by rdlrt** [comment]
@intertreeJK - Please check linked PR
